app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1265,6 +1271,23 @@ public void recenterMap(fr.free.nrw.commons.location.LatLng curLatLng) {
         mapBox.animateCamera(CameraUpdateFactory.newCameraPosition(position), 1000);
     }
 
+    private void showEnableLocationDialog() {
+        // This dialog box prompts the user to set the location to High Accuracy
+        DialogUtil.showAlertDialog(getActivity(), "Turn on location?", "Nearby needs location enabled to work properly",
+                "Yes", "No",  this::openLocationSettings, null);
+    }
```

Comment:
These strings should be moved to `strings.xml`. "Yes" and "No" should already be there.

Reply:
Done